app/assets/stylesheets/main.scss

```diff
@@ -0,0 +1,14 @@
+.site {
+  position: relative;
+  min-height: 100%;
+  padding-bottom: 50px;
+}
+.site-footer {
+  position: absolute;
+  bottom: 0px;
+  width: 100%;
+}
+.footer-content {
+  padding: 10px 10px;
+

```

Comment:
why this new line is needed?

Reply:
![image](https://user-images.githubusercontent.com/28151680/47645807-b3924200-db7a-11e8-99fa-1c4246b52760.png)

Reply:
I mean empty line in CSS